src/FontEmphasisInterface.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+
+namespace HelloFresh\Ausraster;
+
+interface FontEmphasisInterface
+{
+    /**
+     * Set whether the font should be displayed in bold.
+     * @param bool $bold
+     * @return FontInterface
+     */
+    public function setBold(bool $bold) : FontInterface;
```

Comment:
I woudnt expose to public interface setters, they doesnt say anything about actual class behaviour, this can have unexpected results when you re-set several times value and finally is not clear why and in which context it happend